app/controllers/administrator_controller.rb

```diff
@@ -1,36 +1,58 @@
 class AdministratorController < ApplicationController
+  before_action :check_if_admin, only: [:show, :update]
+
   def show
     @orders = Order.all.paginate(page: params[:page], :per_page => 10)
-    @title = "orders"
-    check_if_admin
   end
 
   def food_index
-    @title = "foods"
     @foods = Food.all.order(created_at: :desc)
-    check_if_admin
+    respond_to do |format|
+      format.js 
+    end 
   end
 
   def order_index
-    @title = "orders"
-    @orders = Order.all.paginate(page: params[:page], :per_page => 10)
-    check_if_admin
+    @orders = Order.all.paginate(page: params[:page], per_page: 10)
+    respond_to do |format|
+      format.js 
+    end 
   end
 
   def user_index
-    @title = "users"
-    @users = User.all.paginate(page: params[:page], :per_page => 10).order(created_at: :desc)
-    check_if_admin
+    @users = User.all.paginate(
+                               page: params[:page], 
```

Comment:
Indent the first parameter one step more than the start of the previous line.<br>Trailing whitespace detected.